icefall/score_estimator.py

```diff
@@ -0,0 +1,188 @@
+import argparse
+import glob
+import logging
+from pathlib import Path
+from typing import Tuple, List
```

Comment:
Could you have a look at
https://icefall.readthedocs.io/en/latest/contributing/code-style.html
to follow the code style?